samples/1.0.0-beta8/ConsoleApp/project.json

```diff
@@ -9,7 +9,7 @@
         "dnx451": { },
         "dnxcore50": {
             "dependencies": {
-                "System.Console": "4.0.0-beta-beta8"
+                "System.Console": "4.0.0-23427"

```

Comment:
This is still a wrong package. You should change it into `"System.Console": "4.0.0-beta-23409"`
23409 is the current stable package for beta 8